packages/webview_flutter/webview_flutter_platform_interface/lib/v4/src/platform_navigation_delegate.dart

```diff
@@ -9,6 +9,19 @@ import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 
 import 'webview_platform.dart';
 
+/// Signature for callbacks that report a pending navigation request.
+typedef NavigationRequestCallback = FutureOr<NavigationDecision> Function(
+    {required NavigationRequest navigationRequest});
```

Comment:
I think we can make it a positional argument and not named since we want there to only be one argument moving forward.

Reply:
Yes indeed, done.